src/module.ts

```diff
@@ -68,8 +69,8 @@ export default defineNuxtModule<ModuleOptions>({
           }
 
           const rootDomain = ${JSON.stringify(
-            rootDomains
-          )}.find(domain => hostname.endsWith(domain));
+        rootDomains
+      )}.find(domain => hostname.endsWith(domain));
```

Comment:
I would want to lint a bit

```suggestion
            rootDomains
          )}.find(domain => hostname.endsWith(domain));
```

Reply:
Good catch, thanks! That was an unintentional change.